rrfs-test/validated_yamls/templates/obtype_config/amsua_n19.yaml

```diff
@@ -30,14 +30,14 @@
          - localization method: Horizontal Gaspari-Cohn
            lengthscale: 300e3 # orig
        obs bias:
-         input file: "@BIASFILE@"
+         input file: data/obs/amsua_n19.satbias.nc4
          output file: out_amsua_n19.satbias.nc
          variational bc:
            predictors:
            - name: constant
            - name: lapseRate
              order: 2
-             tlapse: &amsua_n19_tlapse "@LAPSEFILE@"
+             tlapse: &amsua_n19_tlapse data/obs/amsua_n19.tlapse.txt
            - name: lapseRate
              tlapse: *amsua_n19_tlapse
```

Comment:
sorry for my basic question here: Are lines 38-40 and lines 41-42 the same? Why we need two similar `lapseRate`  here?

Reply:
@xyzemc Thanks for the clarification. So `lapseRate*lapseRate` will use the same `tlapse.txt` file?

Is it possible to change the second predictor name to `lapseRate*lapseRate` to avoid confusion? Or does it have to be `lapseRate` as mandated by JEDI?

Reply:
Ah, got it. Thanks!